worldedit-core/src/test/java/com/sk89q/worldedit/internal/expression/ExpressionTest.java

```diff
@@ -64,7 +64,7 @@ public void testEvaluate() throws ExpressionException {
         assertEquals(atan2(3, 4), simpleEval("atan2(3, 4)"), 0);
 
         // check variables
-        assertEquals(8, compile("foo+bar", "foo", "bar").evaluate(5, 3), 0);
+        assertEquals(8, compile("foo+bar", "foo", "bar").evaluate(5D, 3D), 0);
```

Comment:
There's no need to change these, with the vararg / timeout separation in place.

Reply:
yea, as mentioned in the commit message, I decided to leave the change for clarity.